core/src/main/java/org/frankframework/core/Adapter.java

```diff
@@ -710,14 +710,18 @@ private static String getAdditionalLogging(final PipeLineSession pipeLineSession
 	 * Receives incoming messages. If an adapter can receive messages through multiple channels, then add a receiver for each channel.
 	 * @ff.mandatory
 	 */
-	@SuppressWarnings("java:S3457") // Cast arguments to String before invocation so that we do not have recursive call to logger when trace-level logging is enabled
+	@SuppressWarnings("java:S3457") // Cast arguments to String before invocation so that we do not have a recursive call to logger when trace-level logging is enabled
 	public void addReceiver(Receiver<?> receiver) {
 		receivers.add(receiver);
 		if (log.isDebugEnabled()) log.debug("Adapter [{}] registered receiver [{}] with properties [{}]", name, receiver.getName(), receiver.toString());
 	}
 
 	/**
-	 * Formatter for errors that can occur in this adapter.
+	 * Set an {@link IErrorMessageFormatter} that will be used to format an error-message when an exception occurs in this adapter.
+	 * If not set, then, when an exception occurs, the adapter will first check the {@link Configuration#setErrorMessageFormatter(IErrorMessageFormatter)}
+	 * to see if a configuration-wide default error message formatter is set and otherwise create an instance of {@link ErrorMessageFormatter} as default.
```

Comment:
```suggestion
	 * to see if a configuration-wide default error message formatter is set and otherwise create a new instance of {@link ErrorMessageFormatter} as default.
```